docs/ide-setup.md

```diff
@@ -108,22 +108,22 @@ You can sign up for the free [JetBrains student license](https://www.jetbrains.c
 
    * Click `Configure → Project Defaults → Project Structure` (or `File → Project Structure` if a project is currently open).
```

Comment:
You missed this

Reply:
can you specify what is missing?

Reply:
Similar to others where you were changing it to **if the project is open**?